.g8/checkboxPage/app/pages/$className$Page.scala

```diff
@@ -0,0 +1,11 @@
+package pages
+
+import models.$className$
+import play.api.libs.json.JsPath
+
+case object $className$Page extends QuestionPage[Set[$className$]] {
```

Comment:
Yeah I am not to sure about `$className$Page` it might make the journeys very hard to follow.